### DISCUSSION TITLE
Disable cursor word highlight

### DISCUSSION THREAD

**Discussion by jsantiagoh** [LazyVim/LazyVim#4922] [Q&A] [answered]
Hi, 

I can't figure out how to disable this feature since it moved to be lsp-based. I'm talking about lazy highlighting the word I'm currently positioned in and every other instance of it, I understand someone may love it but I find it too distracting.

I read https://www.lazyvim.org/plugins/lsp

and creating a file `lua/plugins/lsp.lua` with either

```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      document_highlight = {
        enabled = false,
      },
    },
  },
}
```

or

```lua
return {
  {
    "neovim/nvim-lspconfig",
   opts = function()
      local ret = {
         document_highlight = {
           enabled = true,
         },
       }
       return ret
     end,
  },
}
```

but nothing works, what am I doing wrong?

**Comment by jsantiagoh** [comment]
For anyone interested, removing `~/local/share/nvim` fixed it

**Reply by folke** [reply]
that is not the solution...

**Reply by folke** [reply, marked as answer]
Should be:

```lua
{
  "folke/snacks.nvim",
  opts = { words = { enabled = false } }
}

**Comment by folke** [comment]
```lua
{
  "folke/snacks.nvim",
  opts = { words = { enabled = false } }
}
```

**Reply by OriLiMu** [reply]
how to  change the color of it? 
And how to change color highlight for current content in bracket? Cant find relate info on the doc. And if list all the highlight to check one by one is too much work to do. 

I add the code to the `init.lua` but not work: 
```lua
vim.api.nvim_set_hl(0, 'CursorWord', {
    fg = '#ff0000',
    ctermfg = 196,
    bold = true
})
```



![image](https://github.com/user-attachments/assets/63fb5c20-ef53-4c72-83c1-654c36a0d3ed)
![image](https://github.com/user-attachments/assets/16b3f1c3-0285-40a3-80c8-de39c5cbca1d)

**Comment by jsantiagoh** [comment]
thank you!
